mscore/instrwidget.cpp

```diff
@@ -479,6 +481,137 @@ void InstrumentsWidget::genPartList(Score* cs)
       partiturList->resizeColumnToContents(4);
       }
 
+
+//---------------------------------------------------------
+//   sanitizePartiturListSelection
+//---------------------------------------------------------
+
+void InstrumentsWidget::sanitizePartiturListSelection()
+      {
+      _partiturSelectedItems.clear();
+      if (partiturList->selectedItems().isEmpty() || partiturList->selectedItems().front() == nullptr) {
+            _partiturSelectionType = PartiturSelectionType::NONE_SELECTED;
+            return;
+            }
+
+      // If a parent is selected, actions apply to all children as well. Select them, but keep track of parents.
+      // If children of multiple parents are selected, select all parents (and their children, by previous convention).
+      QSet<QTreeWidgetItem*> selectedParents;
+      bool containsParents = false;
+      for (QTreeWidgetItem* item : partiturList->selectedItems()) {
+            QTreeWidgetItem* parent = item->parent();
+            if (parent == NULL) {
+                  selectedParents.insert(item);
+                  containsParents = true;
+                  }
+            else {
```

Comment:
Unneeded brackets

Reply:
Isn't this _Exception 2_ under Braces on https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules?

Reply:
Whoops! Missed the "if or else part". Ignore any further mention that falls under that category.